packages/visual-editor/src/components/Locator.tsx

```diff
@@ -610,6 +620,20 @@ const LocationCard: CardComponent<Location> = ({
     "TAP_TO_CALL"
   );
 
+  const documentForLocator = {
+    ...location,
```

Comment:
I may have been wrong in my last comment. Is locale set in the results? It probably isn't so we need to use the locale of the current page, so you can use i18n.language.

Edit: I guess we have streamDocument already so you can use the locale from that